test/setup/outputs.tf

```diff
@@ -14,18 +14,30 @@
  * limitations under the License.
  */
 
-output "project_id" {
+output "kms_project_id" {
   value = module.project_ci_kms.project_id
 }
 
-output "autokey_resource_project_id" {
+output "resource_project_id" {
   value = module.autokey_resource_project.project_id
 }
 
-output "autokey_resource_project_number" {
+output "resource_project_number" {
   value = module.autokey_resource_project.project_number
 }
 
+output "bucket_name_prefix" {
+  value = "ci-kms-bucket"
+}
+
+output "bucket_resource_type_selector" {
+  value = "storage.googleapis.com/Bucket"
+}
+
+output "bucket_location" {
+  value = "us-central1"
+}
+
```

Comment:
These values would be better placed directly in the example folder.
It could be at the variable's default, or hardcoded in example's main.tf file (preferred)

`test/setup` shouldn't have configurations that are used specifically for a single example unless there's a good reason for that. By default, `test/setup` should have just the common resources that are used across all tests that live in the repo.

Reply:
removed the unnecessary vars